cluster/gce/gci/configure-helper.sh

```diff
@@ -224,6 +224,11 @@ function create-master-auth {
   fi
   local -r basic_auth_csv="${auth_dir}/basic_auth.csv"
   if [[ -n "${KUBE_PASSWORD:-}" && -n "${KUBE_USER:-}" ]]; then
+    if [[ -e "${basic_auth_csv}" && "${METADATA_CLOBBERS_CONFIG:-false}" == "true" ]]; then
+      sed -i '/,admin,system:masters$/d' "${basic_auth_csv}"
+      # The following is for the legacy form of the password line.
+      sed -i '/,admin$/d' "${basic_auth_csv}"
```

Comment:
Debating about whether to ask for `,admin,admin$`, in case this line mistakenly finds everything that was in the admin group. But, not sure if that case is going to be hit. Also not entirely sure that's how this file works.